resources/views/shelves/show.blade.php

```diff
@@ -9,19 +9,30 @@
     </div>
 
     <main class="card content-wrap">
-        <h1 class="break-text">{{$shelf->name}}</h1>
+
+        <div class="grid half v-center">
+            <h1 class="break-text">{{$shelf->name}}</h1>
+            <div class="text-m-right my-m">
+                @include('partials.sort', ['options' => [
+                    'name' => trans('common.sort_name'),
+                    'created_at' => trans('common.sort_created_at'),
+                    'updated_at' => trans('common.sort_updated_at'),
+                ], 'order' => $order, 'sort' => $sort, 'type' => 'books'])
```

Comment:
Shelves currently have user-defined ordering. There's no option here to make use of the existing, user-defined ordering.